fendermint/vm/topdown/src/voting.rs

```diff
@@ -294,3 +312,63 @@ where
         })
     }
 }
+
+/// Poll the vote tally for new finalized blocks and publish a vote about them if the validator is bonded.
```

Comment:
is a `bonded` validator those who are in the power table, or how are we defining that?

Reply:
Exactly, it's a nomenclature used for example in the Cosmos SDK about [validators](https://docs.cosmos.network/main/build/modules/staking#validator), and other Proof-of-Stake systems as well. 

I'll rename `known_validator` to `is_bonded` to remove all doubt.

Reply:
Renamed it to `has_power`. Bonding might be too much PoS when some subnets want PoA, but power is power :muscle: